File: Feitos em C/2523.c
#include <stdio.h>
int main() {
	char caracters[27];
	while(scanf("%s",&caracters)==1){
		int lamps;
		scanf("%d",&lamps);
		int onLamps,i;
		for(i=0;i<lamps;i++){
			scanf("%d",&onLamps);
			printf("%c",caracters[onLamps-1]);
		}
		printf("\n");
	}
    return 0;
}

File: Feitos em C Sharp/2523.cs
using System;
namespace ConsoleApp17 {
    class Program {
        static void Main(string[] args) {
			while (true) {
				string caracters = Console.ReadLine();
				if (string.IsNullOrEmpty(caracters)) {
					break;
				}
				int lamps = int.Parse(Console.ReadLine());
				string[] onLamps = Console.ReadLine().Split();
				for (int i = 0; i < lamps; i++) {
					Console.Write(caracters.Substring(int.Parse(onLamps[i])-1,1));
				}
				Console.WriteLine();
			}
		}
    }
}